tests/test_commandline_utils.py

```diff
@@ -60,95 +60,64 @@
 from skll.learner import Learner
 from skll.utils.constants import KNOWN_DEFAULT_PARAM_GRIDS
 
-from tests.utils import make_classification_data, make_regression_data
+from . import other_dir, output_dir, test_dir, train_dir
+from .utils import make_classification_data, make_regression_data, unlink
 
 
 _ALL_MODELS = list(KNOWN_DEFAULT_PARAM_GRIDS.keys())
-_my_dir = abspath(dirname(__file__))
 
 
 def setup():
     """
     Create necessary directories for testing.
     """
-    train_dir = join(_my_dir, 'train')
-    if not exists(train_dir):
-        os.makedirs(train_dir)
-    test_dir = join(_my_dir, 'test')
-    if not exists(test_dir):
-        os.makedirs(test_dir)
-    output_dir = join(_my_dir, 'output')
-    if not exists(output_dir):
-        os.makedirs(output_dir)
-    features_dir = join(_my_dir, 'other', 'features')
-    if not exists(features_dir):
-        os.makedirs(features_dir)
+    for dir_path in [train_dir, test_dir, output_dir,
+                     join(other_dir, 'features')]:
+        Path(dir_path).mkdir(exist_ok=True)
 
 
 def tearDown():
     """
     Clean up after tests.
     """
-    test_dir = join(_my_dir, 'test')
-    output_dir = join(_my_dir, 'output')
-    other_dir = join(_my_dir, 'other')
-
-    if exists(join(test_dir, 'test_generate_predictions.jsonlines')):
-        os.unlink(join(test_dir, 'test_generate_predictions.jsonlines'))
-
-    for f in glob(join(output_dir, 'test_generate_predictions.tsv')):
-        os.unlink(f)
-
-    for model_chunk in glob(join(output_dir,
-                                 'test_print_model_weights.model*')):
-        os.unlink(model_chunk)
 
-    for model_chunk in glob(join(output_dir,
-                                 'test_generate_predictions.model*')):
-        os.unlink(model_chunk)
+    for path in [join(test_dir, 'test_generate_predictions.jsonlines'),
+                 join(other_dir, 'summary_file'),
+                 join(other_dir, 'test_filter_features_input.arff')]:
+        unlink(path)
 
-    for model_chunk in glob(join(output_dir,
-                                 'test_generate_predictions_console.model*')):
-        os.unlink(model_chunk)
-
-    for f in glob(join(other_dir, 'test_skll_convert*')):
-        os.unlink(f)
-
-    if exists(join(other_dir, 'summary_file')):
-        os.unlink(join(other_dir, 'summary_file'))
-
-    if exists(join(other_dir, 'test_filter_features_input.arff')):
-        os.unlink(join(other_dir, 'test_filter_features_input.arff'))
-    for ffile in glob(join(other_dir,
-                           'test_join_features*')):
-        os.unlink(ffile)
-
-    for ffile in glob(join(other_dir, 'features', 'features*')):
-        os.unlink(ffile)
+    for glob_pattern in [join(output_dir, 'test_generate_predictions.tsv'),
```

Comment:
Good catch again! This actually maintains the original code:
```python
   for f in glob(join(output_dir, 'test_generate_predictions.tsv')):
        os.unlink(f)
```
But it should be fixed since it's not a glob pattern.